batch7crud-roll9/src/main/java/com/lftechnology/batch7crud/constants/Parameter.java

```diff
@@ -0,0 +1,18 @@
+package com.lftechnology.batch7crud.constants;
+
+/**
+ * Created by sanjay on 1/21/16.
+ */
+public class Parameter {
+
+  public static final String FIRST_NAME = "fname";
+  public static final String MIDDLE_NAME = "mname";
+  public static final String LAST_NAME = "lname";
+  public static final String ADDRESS = "address";
+  public static final String GRADE = "grade";
+
+  public static final String PAGE = "page";
+
+  private Parameter(){}
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this closing curly brace to the next line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ARightCurlyBraceStartLineCheck)